src/test/js/log-karaoke/noStages.js

```diff
@@ -67,5 +69,19 @@ module.exports = {
                 })
             ;
         });
-    }
+    },
+
+    'Check whether a log which exceed 150kb contains a link to full log and if clicked it disappear': function (browser) {
+        const blueRunDetailPage = browser.page.bluePipelineRunDetail().forRun('noStages', 'jenkins', 1);
+        // first turn on xpath to get the nodes we want
+        browser.useXpath();
+        var xPath = '//div[@class="logConsole"][14]';
```

Comment:
Yeah row 14, that is not general at all, I agree to move common stuff into the pages, but I need to make it more global then.